app/src/test/java/org/mozilla/fenix/sync/ext/ErrorTypeKtTest.kt

```diff
@@ -36,37 +38,22 @@ class ErrorTypeKtTest {
     }
 
     @Test
-    fun `get error item`() {
+    fun `get SyncedTabsListItem_Error item`() {
```

Comment:
The `SyncedTabsListItem_Error` naming threw me off to me while just looking at the `toSyncedTabsListItem` call and I get you meant `SyncedTabsListItem.Error`

I would almost prefer seeing the 'GIVEN WHEN THEN' structure we use: `WHEN toSyncedTabsListItem is called THEN return the correct Error item` It would probably also help that we test that `errorItem.errorType` also returns the correct `SyncedTabsView.ErrorType`